src/data_provider/src/packages/packageLinuxParserApk.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ * Wazuh SYSINFO
+ * Copyright (C) 2015, Wazuh Inc.
+ * December 19, 2022.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "json.hpp"
+#include <fstream>
+
+void getApkInfo(const std::string& fileName, std::function<void(nlohmann::json&)> callback)
+{
+    std::ifstream apkDb(fileName);
+    const size_t valuePos {2};
+    std::string line;
+    std::vector<std::string> data;
+
+    if (apkDb.is_open())
```

Comment:
There's no message log implementation in any of these submodules of data providers, I didn't find any easy way to implement it. Probably it would be desirable to address it in another issue if you consider it important.

Reply:
I wasn't aware of this.

>  Probably it would be desirable to address it in another issue if you consider it important.

We can ask to @Dwordcito and see what is his opinion.

Reply:
It is as Miguel says, here we do not have logs.